hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/ConversionUtils.java

```diff
@@ -66,6 +67,20 @@ public static List<ContractLoginfo> pbjLogsFrom(@NonNull final List<Log> logs) {
         return pbjLogs;
     }
 
+    /**
+     * Wraps the first 32 bytes of the given SHA-384 {@link com.swirlds.common.crypto.Hash hash} in a Besu {@link Hash}.
+     *
+     * @param sha384Hash the SHA-384 hash
+     * @return the first 32 bytes as a Besu {@link Hash}
+     */
+    public static org.hyperledger.besu.datatypes.Hash ethHashFrom(
```

Comment:
(Java's inability to let you define type aliases is sad.  The other time you really want it is for naming a container of some complicated type.)